.changeset/slow-bats-slide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@guardian/cdk": minor
+---
+
+Addition of new deployment mechanism support for MAPI.
```

Comment:
```suggestion
Addition of new experimental deployment mechanism support for MAPI.
```

Just trying to note that usage brings risk (more from it being unused in production thus far).